### PR TITLE
Tribute_Page

### PR DESCRIPTION
**Pull request by faridafaqiri**
**This project is about making a tribute page**
In this project I did this thing:
- First of all I follow GITHUB flow.
- I created HTML file and CSS.
- Linked HTML file with CSS file.
- Added linters config.
- I added main tag, h1 and h3, p, img, blockquote tag and some other tags for creating my web page.
- I added some property in CSS file for styling and designing.
 
tribute_page_test

### CODE REVIEW

File: tribute.html
Comment (ahzamir) [inline tribute.html:1]: Your codes are not alligned correctly and are not clean try to make their spacing right.
- [**Suggession**] - On your HTML file, at the bottom right side of your vscode click on `Spaces`  then click on `Index using spaces`, and then select `2` from there. Then align your components according to the 2 spaces.
- Example:
```
<div>
  <p>...</p>
  <h1>.... </h1>
  <img ... />
</div>
```

**Review by ahzamir — CHANGES_REQUESTED** [review]
### Hi @faridafaqiri 🙋‍♀️,

> Excellent work! You have done a great job implementing the project requirements but your project still misses some of the requirements:

- Your HTML file code is not clean:
- **Check the inline comments below.**

# Status: Required Changes 🔁
Make changes to your code and commit and request for a re-review.

BbcanBbcan3GIF


 ## Highlights 🌞
 
🎯 Great Design implemented.
image

✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed and completed the Tribute Page project task and all requirements.
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
❌ Your Code is clean
**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

File: tribute.html
Comment (faridafaqiri) [reply tribute.html:1]: @ahzamir 
Thank you for feedback!
I cleaned my html code.

**Review by ahzamir — APPROVED** [review]
### Hi @faridafaqiri 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text on it with a screenshot.

- Well Done great design Implemented. 🎯🎉👍
Screenshot 2024-05-23 005526

# Status: Approved ✅

There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳
ExcellentNiceGIF

 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Tribute Page project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

**Comment by faridafaqiri** [comment]
Thank you sir for approving my project.💫

**Pull request merged**